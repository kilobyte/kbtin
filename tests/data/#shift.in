#shift
#ali x {#show {« [$0] [$1] [$2] [$3] [$4] [$5] [$6] [$7] [$8] [$9]};#shift $n;#show {» [$0] [$1] [$2] [$3] [$4] [$5] [$6] [$7] [$8] [$9]}}
#init n
x
x foo bar baz
#var n 3
x
x foo bar baz
x {a b} a {a {b} c} d e
#var n -1
x
#var n dupa
x
