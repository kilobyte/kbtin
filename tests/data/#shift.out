#Error: #shift: no positional variables in this context.
« [] [] [] [] [] [] [] [] [] []
» [] [] [] [] [] [] [] [] [] []
« [foo bar baz] [foo] [bar] [baz] [] [] [] [] [] []
» [bar baz] [bar] [baz] [] [] [] [] [] [] []
« [] [] [] [] [] [] [] [] [] []
» [] [] [] [] [] [] [] [] [] []
« [foo bar baz] [foo] [bar] [baz] [] [] [] [] [] []
» [] [] [] [] [] [] [] [] [] []
« [{a b} a {a {b} c} d e] [a b] [a] [a {b} c] [d] [e] [] [] [] []
» [d e] [d] [e] [] [] [] [] [] [] []
« [] [] [] [] [] [] [] [] [] []
#Error: #shift takes a positive number, got {-1}
» [] [] [] [] [] [] [] [] [] []
« [] [] [] [] [] [] [] [] [] []
#Error: #shift takes a positive number, got {dupa}
» [] [] [] [] [] [] [] [] [] []
